package.json

```diff
@@ -28,16 +29,17 @@
         "react-dom": "^17.0.1",
         "react-google-maps": "^9.4.5",
         "react-icons": "^4.2.0",
-        "react-router-dom": "^5.2.0",
-        "react-scripts": "^4.0.3",
+        "react-router-dom": "^5.3.4",
         "react-transition-group": "^4.4.1",
+        "vite": "^4.3.9",
+        "vite-plugin-svgr": "^3.2.0",
+        "vite-tsconfig-paths": "^4.2.0",
         "web-vitals": "^1.1.1"
     },
     "scripts": {
-        "start": "react-scripts --openssl-legacy-provider start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject"
+        "start": "vite",
+        "build": "tsc && vite build",
```

Comment:
we are not using typescirpt in our project neither you installed typescript ... so remove the tsc fromt the command or build will fail